scripts/e2etests/Utils.ps1

```diff
@@ -75,7 +75,11 @@ function SetRegistryKey
             return $false
         }
 
-        New-Item -Path $RegKey -Force | Out-Null
```

Comment:
Fixed a bug where each call to `SetRegistryKey(…)` would recreate the key.

When SetupFunctionalTests.ps1 would try to set `DoNotShowPreviewWindow` and find the value was not already `1`, `SetRegistryKey(…)` would delete the key and create a new one before setting the value `DoNotShowPreviewWindow`.

Then when SetupFunctionalTests.ps1 would try to set `SuppressUILegalDisclaimer`, it would again delete the key and create a new one before setting the value `SuppressUILegalDisclaimer`.

Thus, the registry key only ever contained the last value set.